test/common/markdown/markdown-spell.tests.ps1

```diff
@@ -0,0 +1,79 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+Describe "Verify Markdown Spelling" {
+    BeforeAll {
+        if(!(Get-Command -Name 'mdspell' -ErrorAction SilentlyContinue)) {
+            start-nativeExecution {
+                sudo npm install -g markdown-spellcheck@0.11.0
+            }
+        }
+
+        # Cleanup jobs for reliability
+        get-job | remove-job -force
+    }
+
+    AfterAll {
+        # Cleanup jobs to leave the process the same
+        get-job | remove-job -force
+    }
+
+    $groups = Get-ChildItem -Path "$PSScriptRoot\..\..\..\*.md" -Recurse | Where-Object {$_.DirectoryName -notlike '*node_modules*'} | Group-Object -Property directory
+
+    $jobs = @{}
+    # Start all spelling verification in parallel
+    Foreach($group in $groups)
+    {
+        Write-Verbose -verbose "starting jobs for $($group.Name) ..."
+        $job = Start-ThreadJob {
+            param([object] $group)
+            foreach($file in $group.Group)
+            {
+                $results = mdspell --en-us --ignore-numbers --ignore-acronyms --report $file 2>&1
+                Write-Output ([PSCustomObject]@{
+                    file = $file
+                    results = $results
+                })
+            }
+        } -ArgumentList @($group)
+        $jobs.add($group.name,$job)
+    }
+
+    # Get the results and verify
+    foreach($key in $jobs.keys)
```

Comment:
This is precarious. If there is ever a folder named `keys` in the repo, that will break this logic. A hashtable with a key called `keys` will return the value of that key instead of all the keys in the hashtable. I do wish they'd fix that, but... 🤷‍♂️ 

Instead:
```suggestion
    foreach($key in $jobs.GetEnumerator().Name)
```